site/docs/tools/spark.md

```diff
@@ -374,11 +374,15 @@ SparkSession.builder
 
 The Nessie LogStore needs the following parameters set in the Spark/Hadoop config.
 
+Note: "BASIC" authentication is not supported by Nessie servers in "production" mode.
+
 ```
 nessie.url = full url to nessie
-nessie.username = username if using basic auth, omitted otherwise
```

Comment:
Maybe I wouldn't remove that section completely but I will just add a note that these configs are being deprecated.

Reply:
I'm not sure I understand, sorry. This is not a removal, but a rename of properties... the new names are mentioned in the "added" lines below.